substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -2024,6 +2024,9 @@ def native_image_on_jvm(args, **kwargs):
             args.append("-D" + key + "=" + value)
 
     arg = [executable]
+    jdk_args = get_jdk().processArgs([])
+    for cur_arg in jdk_args:
+        args.insert(0, '--vm.' + cur_arg[1:])
```

Comment:
This is a neat hack, just afraid this accidentally pulls in more options that are not working as expected. Instead, we could just do what `add_debug_args()` does within mx. We will follow up

Reply:
I updated the code just now. Current code only adds the debug arguments.

Maybe we should mark `add_debug_args()` as not local method so that it can be re-used. But such refactor is better to be done at another PR. What do you think about it?

Reply:
Thanks! I actually just [mirrored your PR](https://github.com/oracle/graal/pull/8861) internally and pushed a similar commit: https://github.com/oracle/graal/pull/8861/commits/1cc9f382a1be175ea3e1c81e82280113eb93d97a

Reply:
> Thanks! I actually just [mirrored your PR](https://github.com/oracle/graal/pull/8861) internally and pushed a similar commit: [1cc9f38](https://github.com/oracle/graal/commit/1cc9f382a1be175ea3e1c81e82280113eb93d97a)

Thanks. I am not familiar to the dev flow of Graal now, but it is strange to has two similar PRs.

Reply:
No problem. We need to mirror external contributions in our internal infrastructure and because I added a commit, the two branches have diverged so the mirror created another PR. Are you ok with my changes in https://github.com/oracle/graal/pull/8861?

Reply:
Ok, great. Thank you! I will take it from here :)